src/main/java/com/powsybl/openloadflow/network/LfAction.java

```diff
@@ -68,6 +70,50 @@ private LoadShift(LfBus bus, String loadId, double p0, PowerShift powerShift) {
         }
     }
 
+    public static final class GeneratorUpdates {
```

Comment:
@Hadrien-Godard and me have exchanged about this feature. We think that it is more complex that we though. So we agree on a first light PR with only changes in `targetP`, relatively and absolutely. So in the `GeneratorUpdate` (I prefer in singular) it is better to have just what you need. Just throw in the `create` function to say that it is not yet supported.